### PR TITLE
Add navbar option for Phase Constraint Calculator

### LINKED ISSUE #343: Add Phase Constraint tool to ExoCTK webpage toolbar
Currently there is no way to navigate to the Phase Constraint tool through the toolbar on the ExoCTK website.  We should add this.

### PR DESCRIPTION
**Pull request by bourque**
This PR adds an option to navigate to the Phase Constraint Calculator from the navbar in the web app.

Closes #343

### CODE REVIEW

**Comment by bourque** [comment]
@jaymedina Could you review?

**Comment by jaymedina** [comment]
@bourque yep! will look at it by end of today

File: exoctk/exoctk_app/templates/base.html [hunk 1]
```diff
@@ -85,6 +85,7 @@
                         </ul>
                         </li>
                         <li><a href="limb_darkening">Limb Darkening Calculator</a></li>
+                        <li><a href="phase_contraint">Phase Constraint Calculator</a></li>
```
Comment (jaymedina) [inline exoctk/exoctk_app/templates/base.html]: did you miss the `s` in `phase_contraint`?

Comment (bourque) [reply exoctk/exoctk_app/templates/base.html]: Whoops!  Good catch!

**Pull request merged**